### ISSUE TITLE
grpo训练过程中的accuracy reward和format reward一直都是1

### ISSUE THREAD

**Issue by fourierer** [modelscope/ms-swift#3130]
您好，我这边在我自己的数据集上进行grpo训练，数据集格式如下：
`{
    "images": [
        "xxx.jpg"
    ],
    "messages": [
        {
            "role": "user",
            "content": "<image>图中一共有多少物体"
        }
    ],
    "solution": "3"
}`
然后grpo训练过程中发现accuracy reward和format reward一直都是1，比如：
![Image](https://github.com/user-attachments/assets/6a14dbc1-42f8-4540-b969-fec6b6ee862c)
![Image](https://github.com/user-attachments/assets/bb6db369-19c7-459e-a11f-f3540424901c)
我的训练指令如下：
`SIZE_FACTOR=8 MAX_PIXELS=602112 CUDA_VISIBLE_DEVICES=0,1 \
swift rlhf \
    --rlhf_type grpo \
    --model xxx/Qwen2.5-VL-7B-Instruct \
    --reward_funcs accuracy format \
    --train_type lora \
    --lora_rank 8 \
    --lora_alpha 32 \
    --torch_dtype bfloat16 \
    --dataset xxx/xxx_grpo.json \
    --max_completion_length 1024 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 4 \
    --learning_rate 1e-5 \
    --gradient_accumulation_steps 16 \
    --eval_steps 100 \
    --save_steps 100 \
    --save_total_limit 2 \
    --logging_steps 10 \
    --max_length 2048 \
    --output_dir xxx/Qwen2.5-VL-7B_grpo_lora \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --dataset_num_proc 4 \
    --num_generations 4 \
    --temperature 0.9 \
    --system '/mnt/workspace/qwen2.5-vl/ms-swift/examples/train/grpo/prompt.txt'`
然后当我把最后一个参数--system去掉之后，重新进行grpo训练，发现accuracy reward一直都是1，但是format reward一直都是0。
这个结果感觉很奇怪，一般这个grpo训练accuracy reward和format reward应该是从0慢慢增加到1才正常

**Comment by Jintao-Huang** [comment]
你可能需要自定义accuracy

**Comment by zytx121** [comment]
@fourierer 请问Qwen2.5-VL-7B-Instruct 跑GRPO需要多少显存？

**Comment by AnsongLi** [comment]
@Jintao-Huang @fourierer 我想问下{ "images": [ "xxx.jpg" ], "messages": [ { "role": "user", "content": "<image>图中一共有多少物体" } ], "solution": "3" } 这种数据形式 solution对应的答案会作为训练token嘛 还是说这部分只在acc reward环节中做一个规则奖励

**Comment by fourierer** [comment]
我本意是想用solution这个字段作为acc reward的标准答案，不作为训练的token

**Comment by Jintao-Huang** [comment]
solution 只为了计算reward的奖励的
